source/particle/particle_handler.cc

```diff
@@ -508,11 +508,11 @@ namespace aspect
 
             std::sort(neighbor_permutation.begin(),
                       neighbor_permutation.end(),
-                      std_cxx11::bind(&compare_particle_association<dim>,
-                                      std_cxx11::_1,
-                                      std_cxx11::_2,
-                                      std_cxx11::cref(vertex_to_particle),
-                                      std_cxx11::cref(vertex_to_cell_centers[closest_vertex_index])));
+                      [&] (const unsigned int a, const unsigned int b) -> bool
+            {
+              return compare_particle_association(a,b,vertex_to_particle,vertex_to_cell_centers[closest_vertex_index]);
```

Comment:
break the arguments onto a separate line here.

also for the arguments of the lambda